person.rb

```diff
@@ -0,0 +1,32 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
+
+  public
+
+  def can_use_services?
+    true if @age >= 18 || @parent_permission
+  end
+
+  def check_age?
+    of_age?
+  end
+end
+
+# person1 = Person.new(17)
+# puts person1.check_age?
```

Comment:
Please remove commented code from here as it can make the whole codebase unclean.

Reply:
Done ✔

Reply:
This issue has been addressed well. Nice job! :pray: